book/src/specs/kimchi.md

```diff
@@ -1398,6 +1388,86 @@ Kimchi framework that dictated those approaches.
 * only 4 lookups per row
 * only first 7 columns are available to the permutation polynomial
 
+##### `KeccakRot` - Constraints for rotation of 64-bit words
```

Comment:
Since this gate is a general-purpose rotation gate, should we call it `Rot` instead? It will be useful for SHA256 and other hash functions, not just keccak!

Reply:
Ok, so basically the same as in XOR last time. Nice, I will have it as an independent gate and inside the Keccak module only the strictly Keccak operations -> meaning the sequence of gates to build the gadget etcetera